PEReaderAndWriter/PEWriter/PeWriter.cs

```diff
@@ -1001,8 +1001,8 @@ class SuffixSort : IComparer<string> {
       if (this.module.ILOnly) result |= 1;
       if (this.module.Requires32bits) result |= 2;
       if (this.module.StrongNameSigned) result |= 8;
-      if (this.module.Prefers32bits) result |= 0x12;
       if (this.module.TrackDebugData) result |= 0x10000;
+      if (this.module.Prefers32bits) result |= 0x20000;
```

Comment:
I believe this is supposed to be 0x20002 since Requires32bits should get set in the output.

Reply:
Sorry, I'm confused. Why should Requires32bits get set? Wouldn't line 1002 take care of that if that property was set?